js/less/bqplot.less

```diff
@@ -50,7 +50,7 @@
     --bq-selector-fill: var(--jp-border-color1);
 
     --bq-mark-tooltip-border: 2px solid var(--jp-border-color0);
-    --bq-mark-tooltip-box-shadow: 1px 1px 2px rgba(245, 245, 245, 0.8);
+    --bq-mark-tooltip-box-shadow: var(--jp-elevation-z6);
```

Comment:
This seems to be the value used in JupyterLab for box shadows

Reply:
Actually this variable is not there in Jupyter Notebook, so I now use the actual variable value.